packages/core/src/Stores/common-store.js

```diff
@@ -111,7 +111,7 @@ export default class CommonStore extends BaseStore {
             this.is_language_changing = true;
             this.changing_language_timer_id = setTimeout(() => {
                 this.is_language_changing = false;
-            }, 10000);
+            }, 2500);
```

Comment:
This was 10000 because it might break something else. Can you consult @yauheni-deriv on this

Reply:
What is it breaking?

Reply:
I am unable to reproduce the issue specified by @yauheni-deriv